numba/tests/test_parfors.py

```diff
@@ -3346,7 +3350,7 @@ def generate_prange_func(self, pyfunc, patch_instance):
             # look for LOAD_GLOBALs that point to 'range'
             for instr in dis.Bytecode(pyfunc_code):
                 if instr.opname == 'LOAD_GLOBAL':
-                    if instr.arg == range_idx:
+                    if _fix_LOAD_GLOBAL_arg(instr.arg) == range_idx:
```

Comment:
Not for now, but this also needs 6b101e2 from #8621 to address the correct index following the injection of the `prange` into `co_names`.

Reply:
yup, i saw that when i reviewed #8621. keeping this as is in this pr